src/tasks/deploy.mk

```diff
@@ -1,11 +1,22 @@
 #Must be above deplo%
-deploy-asset%: ## deploy-assets: uploads static files such as CSS and JS to S3 based on the contents of a manifest file
-	@if [ -e public/manifest.json ]; then\
-		nht deploy-hashed-assets --monitor-assets --manifest-file manifest.json --assets-are-hashed --destination-directory page-kit;\
+deploy-asset%: ## deploy-assets: uploads static files such as CSS and JS to S3
+	@if [ -e public/manifest.json ]; then \
+		nht upload-assets-to-s3 \
+			--accessKeyId=$(aws_access_hashed_assets) \
+			--secretAccessKey=$(aws_secret_hashed_assets) \
+			--directory="public" \
+			--bucket="ft-next-hashed-assets-prod" \
+			--destination="hashed-assets/page-kit" \
+		&& nht upload-assets-to-s3 \
+			--accessKeyId=$(aws_access_hashed_assets) \
+			--secretAccessKey=$(aws_secret_hashed_assets) \
+			--directory="public" \
+			--bucket="ft-next-hashed-assets-prod-us" \
+			--destination="hashed-assets/page-kit"; \
```

Comment:
Silly Make question, but do you need this last trailing slash? I see it everywhere else so I assume so, it just looks out of place.

Reply:
I _think_ so, I thought this was a bash conditional rather than a Make one so a newline would terminate the statement before the closing tag.